.github/workflows/dotnetcore.yml

```diff
@@ -0,0 +1,17 @@
+name: .NET Core
+
+on: [push]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Setup .NET Core
+      uses: actions/setup-dotnet@v1
+      with:
+        dotnet-version: 2.2.108
```

Comment:
The version might be `3.1.403`, because 2.2 is not supported anymore. 3.1 has LTS.